ui/src/config/section/account.js

```diff
@@ -196,6 +196,11 @@ export default {
       message: 'message.delete.account',
       dataView: true,
       show: (record, store) => {
+        // Don't allow users to delete their account
+        if (record.id !== 'undefined' && store.userInfo.accountid === record.id) {
+          return false
+        }
+
         return ['Admin', 'DomainAdmin'].includes(store.userInfo.roletype) && !record.isdefault &&
```

Comment:
May be disabling the action button makes it better, something like 

disabled: (record, store) => { return (record.id !== 'undefined' && store.userInfo.accountid === record.id) },

Reply:
I agree that disabling the button would be a better option, what do you think @soreana ?

Reply:
@harikrishna-patnala @JoaoJandre Yeah I totally agree, I didn't have time to fix it :D
Now I've applied your suggestion.